tests/plugins/imagebase/features/link.js

```diff
@@ -389,6 +389,63 @@
 					} );
 				}
 			} );
+		},
+		'test link option in context menu': function( editor, bot ) {
+			addTestWidget( editor );
```

Comment:
All tests have nearly the same code. It'd be nice to extract common logic into some helper function (e.g. `createContextMenuTest` or something similar).